### ISSUE TITLE
sonar-web-frontend-plugin Install successfully but run sonar-runner not detect any error

### ISSUE THREAD

**Issue by milaninno** [groupe-sii/sonar-web-frontend-plugin#41]
installed sonar-web-frontend-plugin  but i run sonar-runner but not detect any error.

**Comment by milaninno** [comment]
Please reply urgent basis

**Comment by FridaCai** [comment]
I meet the same problem. Is there any body maintaining the project?

**Comment by cedric-legallo** [comment]
Maybe :smile: 
The main contributor stills follow the repository but does not have much time to develop on it.
By the way this issue does not give any context to help the contributor answer to it.
I personnally installed this plugin on sonar 5.6, 2 months ago and had no problem (except for the duplication not working)
The activity here is low, so if you want answers, be precise ! (or better : contribute since the main contributor searchs other contributors to help him)

**Comment by oxie** [comment]
Guyz this is really cool plugin, please find a way to fix it. Our company needs it as there isnt any good alternative for it.

**Comment by sree526** [comment]
Even for me it's not working I'm using eslint-angular as default profile and it doesn't show any errors. I have activated so many rules and uploaded some buggy code still no buys show up.. Help me out

**Comment by DavidGDD** [comment]
I've the same problem. I'm using sonarqube 6.7 and sonar-scanner-3.0.3. it seems to work well setting new analysis in sonarqube, but no errror included in the analysis.

This is the sonar-scanner log:

```
INFO: Scanner configuration file: .../sonar-scanner-3.0.3.778-linux/conf/sonar-scanner.properties
INFO: Project root configuration file: .../pruebaSonar/sonar-project.properties
INFO: SonarQube Scanner 3.0.3.778
INFO: Java 1.8.0_121 Oracle Corporation (64-bit)
INFO: Linux 4.13.0-17-generic amd64
INFO: User cache: /root/.sonar/cache
INFO: Publish mode
INFO: Load global settings
INFO: Load global settings (done) | time=58ms
INFO: Server id: AWAIg1tKGzwVLOTG9G8a
INFO: User cache: /root/.sonar/cache
INFO: Load plugins index
INFO: Load plugins index (done) | time=84ms
INFO: SonarQube server 6.7.0
INFO: Default locale: "es_ES", source code encoding: "UTF-8"
INFO: Process project properties
INFO: Load project repositories
INFO: Load project repositories (done) | time=66ms
INFO: Load quality profiles
INFO: Load quality profiles (done) | time=71ms
INFO: Load active rules
INFO: Load active rules (done) | time=2566ms
INFO: Load metrics repository
INFO: Load metrics repository (done) | time=109ms
INFO: Project key: NVM
INFO: -------------  Scan NUEVO
INFO: Load server rules
INFO: Load server rules (done) | time=838ms
INFO: Base dir: .../pruebaSonar
INFO: Working dir: .../pruebaSonar/.scannerwork
INFO: Source paths: .
INFO: Source encoding: UTF-8, default locale: es_ES
INFO: Index files
INFO: 5 files indexed
INFO: Quality profile for js: All linters
WARN: The report file .../pruebaSonar/reports/sonar/js-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/eslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/scss-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/html-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/angular-hint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/csslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/eslint-angular.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/tslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-overall.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/css-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/htmlhint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-it.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-it.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-ut.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-ut.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/scsslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-it.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-ut.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-it.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-overall.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/jshint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-ut.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/eslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/scss-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/html-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/angular-hint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/csslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/eslint-angular.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/tslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-overall.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/css-duplication.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/htmlhint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-it.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-it.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-ut.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-ut.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/scsslint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-it.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-ut.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-it.xml doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/js-overall.lcov doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/jshint.json doesn't exist
WARN: The report file .../pruebaSonar/reports/sonar/ts-ut.xml doesn't exist
INFO: Sensor SonarJavaXmlFileSensor [java]
INFO: Sensor SonarJavaXmlFileSensor [java] (done) | time=0ms
INFO: Sensor Analyzer for "php.ini" files [php]
INFO: Sensor Analyzer for "php.ini" files [php] (done) | time=3ms
INFO: Sensor Zero Coverage Sensor
INFO: Sensor Zero Coverage Sensor (done) | time=11ms
INFO: Sensor CPD Block Indexer
INFO: Sensor CPD Block Indexer (done) | time=0ms
INFO: Calculating CPD for 0 files
INFO: CPD calculation finished
INFO: Analysis report generated in 81ms, dir size=39 KB
INFO: Analysis reports compressed in 10ms, zip size=11 KB
INFO: Analysis report uploaded in 23ms
INFO: ANALYSIS SUCCESSFUL, you can browse http://localhost:9000/dashboard/index/NVM
INFO: Note that you will be able to access the updated dashboard once the server has processed the submitted analysis report
INFO: More about the report processing at http://localhost:9000/api/ce/task?id=AWAIoCwgkPje4vqd_t8g
INFO: Task total time: 5.037 s
INFO: ------------------------------------------------------------------------
INFO: EXECUTION SUCCESS
INFO: ------------------------------------------------------------------------
INFO: Total time: 6.432s
INFO: Final Memory: 52M/279M
INFO: ------------------------------------------------------------------------
```

Obviusly the problem is in the reports, but i don't understand what is the problem who produces this behaviour.
